Product.md

```diff
@@ -2,32 +2,117 @@
 
 ## Product Manager
 
-> I understand and advocate for Octopus users, and have a deep knowledge of the ecosystem Octopus lives in.  I help shape the strategic direction for the Octopus product. I am passionate about building software that users love. 
+> I am curious how our users and customers solve problems for my product area(s). Others seek me out as the person who is thinking about where we could go next. I use our constraints and insights to prioritize hard problems that our customers will value, solving in a way that works for our business. I work with peers and stakeholders to remove barriers to meaningful progress. 
 
-### I understand and advocate for Octopus users
+### I discover the needs and problems that matter most to our customer
 
-- I spend time listening to users, through various communication channels  
-- I take the time to understand the problems they are trying to solve 
-- I use data effectively to predict and report the success of delivered features 
-- I use the insight I gain about our users to improve the Octopus product 
+- I discuss with customers, regularly, their needs and desires to identify opportunities to improve our product
+- I discuss with customer teams emerging trends and opportunities to improve my product area(s)
+- I use research and data to inform my perspective of the impact of the opportunities we are considering
+- I monitor key movements of competitors and broader industry trends to maintain our competitive advantages
 
-### I understand the ecosystem Octopus lives in 
+### I set the direction for my product area(s)
 
-- I have a deep working knowledge of the software development lifecycle 
-- I spend time evaluating other tools in the ecosystem
-- I understand Octopus's relative strengths and opportunities
-- I use the insight I gain about our competitors to improve the Octopus product 
+- I influence how others think about the direction for my product area(s) to create interest that helps me refine the direction
+- I collect a range of inputs and perspectives to inform the decisions I make in shaping opportunities
+- I evaluate, clarify, and prioritize the opportunities that we are pursuing to create focus and progress for my product area(s)
+- I align the product team and key stakeholders (cross-functional managers) to create support for our choices
 
-### I am recognised for my communication skills   
+### I remove barriers to achieve meaningful progress
 
-- I am an information honey bee, helping all the teams at Octopus connect the dots 
-- I communicate clearly, in the right places, at the right times. That probably means I communicate a _lot_!
-- I am passionate and persuasive when discussing product direction 
-- I help the marketing team to tell a compelling story
+- I define the opportunity we are pursuing and why we should do it now to ensure that all contributors can explain it themselves
```

Comment:
What is the scope of the opportunities as defined here? For example, the "Opportunities and Solutions" diagram at https://whimsical.com/opportunities-and-solutions-ARZnEzmgQFvxrNyGPYo36n lays out a reasonably clear set of opportunities that a product manager would work within.

Reply:
As discussed, keep watching this diagram through the next week or two for clarity to emerge.